tests/post/steps/test_service_configuration.py

```diff
@@ -63,15 +54,14 @@ def check_csc_configuration(
     name,
     namespace,
 ):
-    csc_response = csc.get(name, namespace)
+    csc_obj = csc.get(name, namespace)
 
-    assert csc_response, (
+    assert csc_obj, (
         "No ConfigMap with name {} in namespace {} found".format(
             name, namespace
         )
     )
 
-    csc_obj = csc.load(csc_response, name, namespace)
     return dict(csc_obj=csc_obj)
```

Comment:
Why do we return here? And why wrap it in a higher-level dict? (not your code again :/ but since you're refactoring...)

Reply:
Actually, could be smart to return the `csc_obj` as a fixture and consume it from the `when`. And change the `ClusterServiceConfiguration` class to manage a single instance of CSC (not just a client), giving it methods to handle teardown at different fixture scopes... For the future ;)

Reply:
Agree, just trying to change as less as possible as it's not related to the ticket linked to this PR.